pkg/constants/metadata.go

```diff
@@ -14,15 +14,24 @@ package constants
 
 // Constants that are used in labels and annotation on DevWorkspace-related resources.
 const (
-	// DevWorkspaceIDLabel is label key to store workspace identifier
+	// DevWorkspaceIDLabel is the label key to store workspace identifier
 	DevWorkspaceIDLabel = "controller.devfile.io/devworkspace_id"
 
 	// DevWorkspaceCreatorLabel is the label key for storing the UID of the user who created the workspace
 	DevWorkspaceCreatorLabel = "controller.devfile.io/creator"
 
-	// DevWorkspaceNameLabel is label key to store workspace name
+	// DevWorkspaceNameLabel is the label key to store workspace name
 	DevWorkspaceNameLabel = "controller.devfile.io/devworkspace_name"
 
+	// DevWorkspaceMountLabel is the label key to store if a configmap or secret should be mounted to the devworkspace
+	DevWorkspaceMountLabel = "controller.devfile.io/mount-to-devworkspace"
+
+	// DevWorkspaceMountPathAnnotation is the label key to store the mount path for the secret or configmap
+	DevWorkspaceMountPathAnnotation = "controller.devfile.io/mount-path"
+
+	// DevWorkspaceMountAsAnnotation is the label key to store if the configmap or secret should be mounted as a file or environment variables
```

Comment:
please consider adding expected possible values. I guess it's `env`/`file`

Reply:
done!